client/checkout/api/index.js

```diff
@@ -279,61 +279,56 @@ export default class WCPayAPI {
 			);
 		};
 
-		const confirmAction = confirmPaymentOrSetup();
```

Comment:
The changes here might look scary, but if you remove the whitespace they become a bit more legible: https://github.com/Automattic/woocommerce-payments/pull/8091/files?w=1